report/report.go

```diff
@@ -50,290 +62,15 @@ func GenerateReport(r *detector.DetectionResults, directory string) string {
-						C261.649,444.996,260.826,445.136,260.004,445.136z M178.899,275.002c-2.088,0.374-3.623,2.169-3.623,4.291v67.011
-						c0,46.426,65.884,76.177,83.094,83.153c1.048,0.425,2.212,0.424,3.26-0.001c17.197-6.989,83.094-36.806,83.094-83.151v-67.011
-						c0-2.121-1.534-3.917-3.623-4.291c-22.015-3.946-24.654-19.961-24.654-27.384c0-0.406,0.007-0.811,0.02-1.216
-						c0.072-2.153-1.503-4.019-3.628-4.37c-15.394-2.536-33.709-3.905-52.836-3.905c-19.13,0-37.446,1.368-52.836,3.904
-						c-2.125,0.35-3.7,2.217-3.628,4.37c0.014,0.404,0.02,0.81,0.02,1.216C203.56,255.041,200.919,271.057,178.899,275.002z"/>
-					<path d="M301.097,296.658h-10.091v-17.706c0-17.096-13.91-31.006-31.006-31.006c-17.096,0-31.006,13.91-31.006,31.006v17.706
-						h-10.091c-4.007,0-7.26,3.252-7.26,7.26v82.194c0,4.007,3.252,7.26,7.26,7.26h82.194c4.007,0,7.26-3.252,7.26-7.26v-82.194
-						C308.357,299.91,305.104,296.658,301.097,296.658z M243.513,278.951c0-9.089,7.398-16.487,16.487-16.487
-						c9.089,0,16.487,7.398,16.487,16.487v17.706h-32.973V278.951z M293.837,373.044c0,3.209-2.599,5.808-5.808,5.808H231.97
-						c-3.209,0-5.808-2.599-5.808-5.808v-56.059c0-2.12,1.133-3.978,2.831-4.987c0.871-0.523,1.888-0.82,2.976-0.82h56.059
-						c1.089,0,2.105,0.298,2.977,0.82c1.699,1.009,2.831,2.868,2.831,4.987V373.044z"/>
-					<path d="M265.65,348.203c3.706-2.009,6.222-5.932,6.222-10.444c0-6.557-5.315-11.872-11.872-11.872
-						c-6.557,0-11.872,5.315-11.872,11.872c0,4.511,2.517,8.435,6.222,10.444l-5.099,11.889c-0.822,1.916,0.584,4.048,2.669,4.048h16.16
-						c2.085,0,3.491-2.132,2.669-4.048L265.65,348.203z"/>
-				</g>
-				</svg>
-			</div>
-
-			<h1 id="heading">Talisman Scan Report</h1>
-			<div class="scan-report">
-				<table class="report-table">
-					<tr>
-						<th id="file-path">File Path</th>
-						<th>
-							<span id="message-heading">Message</span>
-							<span id="commit-heading">Found in Commit (Git SHAs)</span>
-						</th>
-					</tr>
-					{{range $filePath, $FailureData := .Failures}}
-						<tr>
-							<td>{{$filePath}}</td>
-							<td>
-								<table class="details">
-									{{range $failureMessage, $commits := $FailureData.FailuresInCommits}}
-										<tr>
-											<td class="failure-message">
-												{{$failureMessage}}
-											</td>
-											<td>
-												<table>
-												{{range $commit := $commits}}
-													<tr><td>{{$commit}}</td></tr>
-												{{end}}
-												</table>
-											</td>
-										</tr>
-									{{end}}
-								</table>
-							</td>
-						</tr>
-					{{end}}
-				</table>
-			</div>
-			<footer class="site-footer">
-  				<div class="wrapper">
-    				<h3 class="footer-heading"><a href="https://github.com/thoughtworks/talisman/">GitHub</a></h3>
-    				<h2 class="footer-heading">&#9400; 2016&nbsp;ThoughtWorks, Inc.</h2>
-  				</div>
-			</footer>
-		</body>
-	</html>`
+func generateErrorMsg() {
+	color.HiMagenta("\nLooks like you are using 'talisman --htmlreport' for scanning.")
```

Comment:
this should say --scanWithHtml instead of --htmlReport

Reply:
Done